sdk/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/model/response/RefundDetailsResponse.java

```diff
@@ -19,9 +23,9 @@ public class RefundDetailsResponse {
     private final String createdAt;
     private final String updatedAt;
     private final Money refundMoney;
-    private final String vatCategory;
-    private final String paymentBrand;
-    private final String status;
+    private final VatCategory vatCategory;
+    private final PaymentBrand paymentBrand;
+    private final TransactionStatus status;
```

Comment:
I would be interested to have the opinion of @mout-rabobank on this.  Would it be better to leave this as String for backward compatibility? Would it be conceivable that we add a new paymentbrand but the merchant still has an old version of the sdk which will cause the enum conversion to fail here?